src/table/components/TableHeadWrapper.jsx

```diff
@@ -42,6 +42,11 @@ function TableHeadWrapper({
               setFocusedCellCoord,
             });
           };
+          const handleMouseDown = (evt) => {
+            evt.preventDefault();
+            const cell = getCellElement(rootElement, [columnIndex, 0]);
+            updateFocus({ focusType: 'focus', cell });
```

Comment:
what happens if all this function does is preventDefault? Does the event bubble to onMouseDown on the cell?

Reply:
Yes, the event bubbles further up the chain as evt.stoppropagation() is intentionally not used.

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#fixing_the_problem_with_stoppropagation

Reply:
so then do you need to to anything else here than preventDefault?

Reply:
no, I do not think so.

Reply:
I tried and it doesn't for some weird reason, might be some other listener by deafult on the mui component. anyway, just use `handleClickToFocusHead` here too. it should do the same thing

Reply:
[columnIndex, 0] the order is wrong, should be [0, columnIndex].  handleClickToFocusHead also calls when clicking label.

Reply:
just use handleClickToFocusHead here does not work.